### DISCUSSION TITLE
If i want to scale my temporal system, workflow execution capabiltiy, what should i scale, my worker or my temporal server

### DISCUSSION THREAD

**Discussion by manojkumargogula** [alexandrevilain/temporal-operator#717] [Q&A]
@alexandrevilain  I have a doubt in scaling up the temporal scaling mechanism, if i want to scale my temporal executions, what should i scale, if the worker is taking care of executions, why should i need to scale my temporal

**Comment by alexandrevilain** [comment]
Hi! 

Maybe you could check the following documentations:
- https://docs.temporal.io/self-hosted-guide/production-checklist
- https://mikhail.io/2021/05/choose-the-number-of-shards-in-temporal-history-service/
- https://docs.temporal.io/clusters

Please note this tool is not an official temporal product. Consider asking this kind of questions to the:
- Temporal community forum
- Temporal slack
